wlib/SourceView.cpp

```diff
@@ -469,6 +458,29 @@ void CSourceCtrl::DoInit()
     SetFoldFlags(SC_FOLDFLAG_LINEAFTER_CONTRACTED);
 }
 
+void CSourceCtrl::InitLanguage(IAttributeType *attrType, BOOL clearSourceType)
```

Comment:
Do you ever set attrType and have clearSourceType=false?

Reply:
Yes. It is false as a default in the call in SourceView.cpp on line 392.

Reply:
No. I thought of the same thing. But I usually leave the option open. Could go either way.